demo/AudioSin.hs

```diff
@@ -0,0 +1,154 @@
+{-# LANGUAGE MultiWayIf #-}
```

Comment:
This demo is so cool! We should say a few words in the module comment about what it does. I think it also deserves a little blog post because while this isn't the only way someone might want to do such a thing, I am certain that many people would not think of using ffmpeg for this, and you've done it so nicely. I'd be happy to write a short note about it if you're not interested.